seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -695,7 +695,7 @@ def _assert_valid_operation(self, project, test_add_data=True):
         if test_add_data:
             updated_anvil_variables = ADD_DATA_UPDATED_ANVIL_VARIABLES
             genome_version = 'GRCh37'
-            temp_file_data = b's\nHG00731\nHG00732\nHG00733\nHG00735\nNA19675\nNA19675_1\nNA19678\nNA19678\nNA20870\nNA20874'
+            temp_file_data = b's\nHG00731\nHG00732\nHG00733\nHG00735\nNA19675\nNA19675_1\nNA19678\nNA20870\nNA20874'
```

Comment:
there was actually a bug with a duplicated ID - NA19678 was included twice and is now only included once